src/lens.js

```diff
@@ -7,8 +7,11 @@ export { default as compose } from 'ramda/src/compose';
 
 import lens from 'ramda/src/lens';
 import view from 'ramda/src/view';
+import lset from 'ramda/src/set';
```

Comment:
Why did you call this `lset` instead of just set?

Reply:
Hahaha, this is the source of almost 2. hours of debugging.... It's because it conflicted with https://github.com/microstates/microstates.js/blob/050e2a2bfc9a28267d299f99cc5480363ba99a62/src/lens.js#L21 yet weirdly kinda worked in some situations.... so the error was not obvious.

Definitely a case where a static type system would have raised the issue immediately by recognizing that the two `set` functions had different signatures. HashTagLaughCry